web/frontend/src/components/slam/index.svelte

```diff
@@ -278,13 +277,21 @@ const toggleAxes = () => {
 
 const handleMoveClick = async () => {
   try {
+    // set pose in frame
+    const lastPose = await slamClient.getPosition();
```

Comment:
Just curious, why do we need to fetch the last pose if we're overwriting the `xy` _and_ specifying `position_only` when calling `moveOnMap`? My naive assumption would be that rotation wouldn't be needed to make this call.

Reply:
> Just curious, why do we need to fetch the last pose if we're overwriting the `xy` _and_ specifying `position_only` when calling `moveOnMap`? My naive assumption would be that rotation wouldn't be needed to make this call.

As motion.MoveOnMap is currently implemented: the rotation is needed for motion planning to plan correctly. For example, if we didn't do this then oZ would be undefined, which I assume would become 0 when sent over the wire, which would be an invalid orientation. 

Also, if we were not specifying `position_only_mode`, the destination orientation information would prescribe the ending orientation the user wants the base to have (described in theta).

Reply:
Got it, thanks for satisfying my curiosity!